packages/remix-dev/config.ts

```diff
@@ -371,22 +371,22 @@ export async function readConfig(
   let routes: RouteManifest = {
     root: { path: "", id: "root", file: rootRouteFile },
   };
+  function addRoutes(data: RouteManifest) {
+    for (let key of Object.keys(data)) {
+      let route = data[key];
+      routes[route.id] = { ...route, parentId: route.parentId || "root" };
+    }
+  }
```

Comment:
While we're at it, let's make this

```suggestion
  let addRoutes = (routes: RouteManifest) =>
    Object.values(routes).forEach((route) => {
      routes[route.id] = { ...route, parentId: route.parentId || "root" };
    });
```